app/src/main/java/com/example/trabalhador/util/KeyboardUtils.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.trabalhador.util
+
+import android.content.Context
+import android.view.View
+import android.view.inputmethod.InputMethodManager
+import androidx.fragment.app.Fragment
+
+object KeyboardUtils {
+    fun hideKeyboard(fragment: Fragment) {
+        val imm = fragment.requireActivity().getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
```

Comment:
Não é pra acontecer, mas só por precaucao, podemos adicionar um `as?` pra garantir que se for `null` não quebra o app hehe